setup/web_server_configuration.rst

```diff
@@ -31,6 +31,22 @@ to use PHP :ref:`with Nginx <web-server-nginx>`.
 
 .. _web-server-apache-mod-php:
 
+Adding Rewrite Rules
+--------------------
+
+The easiest way is to install the apache recipe by executing the following command:
+
+.. code-block:: terminal
+
+    $ composer require symfony/apache-pack
+
+This recipe will install the ``.htaccess`` file in the ``public/`` directory.
+
+.. tip::
+
+    A performance improvement can be achieved by putting the rewrite rules in your
```

Comment:
Suggestion:
"by moving the rewrite rules from the `.htaccess` file to the `VirtualHost` block of your Apache configuration and disallowing `.htaccess` overrides by changing ``AllowOverride All`` to ``AllowOverride None`` (where though?).